coops-ui/src/mixins/date-mixin.ts

```diff
@@ -51,4 +51,15 @@ export default class DateMixin extends Vue {
 
     return eval(date1 + operator + date2) // eslint-disable-line no-eval
   }
+
+  /**
+   * Formats a simple date string (YYYY-MM-DD) to (Month Day, Year) for readability.
+   *
+   * @param date The date string to format.
+   * @returns The re-formatted date string without the day name.
+   */
+  toReadableDate (date: string): string {
+    const regex = / (?!.* )/
+    return (new Date(date).toDateString().split(' ').slice(1).join(' ').replace(regex, ', '))
```

Comment:
Severin: This statement is hard to read and understand. If you're updating this file, I recommend you separate the parts of the algorithm, as is done with other functions in this module :)

Cameron: Certainly can break down the string manipulation and regex, absolutely!

Reply:
Better, thanks!